src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -240,6 +254,11 @@ protected function validate($config, string $name, string $path)
         if (!\is_array($config)) {
             throw new \InvalidArgumentException(sprintf('The definition of "%s" in "%s" must be a YAML array.', $name, $path));
         }
+        if (isset($config['alias'])) {
+            $this->validateAlias($config, $name, $path);
+
+            return;
```

Comment:
What's about other checks below?

Reply:
If it's an alias, it doesn't make sense to do the other checks.